test/account/all_resources_integration_test.go

```diff
@@ -49,7 +51,10 @@ func TestDeployAndDelete_AllResources(t *testing.T) {
 		myLocalGroup := "My LOCAL Group%RAND%"
 		myPolicy := "My Policy%RAND%"
 		myPolicy2 := "My Policy 2%RAND%"
-		envVkb := "vkb66581"
+		env1, ok := os.LookupEnv("ENVIRONMENT_ID_2")
```

Comment:
could we also name the variable env2, its a bit misleading that we read the ENV variable ID 2 into env 1 :)